README.md

```diff
@@ -350,3 +363,5 @@ still can.
 * Code written in ES2015+
 * Universal rendering via node
 * Unit tests with high coverage (aiming for 100%)
+
+[jest]: http://facebook.github.io/jest/docs/en/getting-started.html
```

Comment:
Should be HTTPS.

![tinfoil-hat](https://cloud.githubusercontent.com/assets/90871/26472602/cf452e30-419e-11e7-8c4d-266c76497175.jpg)